services/venice-router/src/test/java/com/linkedin/venice/router/api/TestVeniceVersionFinder.java

```diff
@@ -275,7 +277,7 @@ public void returnsCurrentVersionWhenTheDictionaryExists() {
     }
   }
 
-  @Test
+  @Test(enabled = false)
```

Comment:
why do we disable it? If we don't need, we can remove it. Is there an issue that keeps failing this test?

Reply:
enabled it